R/calcAGEB.R

```diff
@@ -15,7 +15,7 @@
 calcAGEB <- function(subtype = "balances") {
   ageb <- readSource("AGEB", subtype = subtype)
 
-  mapping <- toolGetMapping("Mapping_AGEB_REMIND.csv", type = "reportingVariables") %>%
+  mapping <- toolGetMapping("Mapping_AGEB_REMIND.csv", type = "reportingVariables", where="mappingfolder") %>%
```

Comment:
Please make sure that your added code does not produce linter warnings. This addition should cause a complaint about missing whitespaces.

Reply:
[Added](https://github.com/pik-piam/mrremind/pull/405/commits/16631c6857f80c78cf27909126dbe3c8a8407076).